Example/Views/ModalScreen.swift

```diff
@@ -44,6 +44,17 @@ enum ModalScreen {
                             textView.textFont = Font(name: "Helvetica")
                             textView.textSize = 18
                         }),
+                    imageView(
+                        image: Image(image: UIImage(named: "test")!),
+                        style: styleSheet {
+                            $0.backgroundColor = .yellow
+                        },
+                        layout: layout {
+                            $0.height = Dimension(value: 50)
+                            $0.width = Dimension(value: 50)
+                            $0.alignment = Alignment(self: .center)
```

Comment:
use

```swift
aligment() {
  $0.`self` = .center
}
```

Reply:
same question as in the previous comment.

Reply:
Yes, all layout objects that have a builder function should have `internal` inits.